src/tagmanager/tm_workspace.c

```diff
@@ -453,19 +419,19 @@ static GList *lookup_includes(const gchar **includes, gint includes_count)
 #ifdef HAVE_GLOB_H
 	globbuf.gl_offs = 0;
 
-	if (includes[0][0] == '"')	/* leading \" char for glob matching */
+	if (sources[0][0] == '"')	/* leading \" char for glob matching */
```

Comment:
I know its not something you changed (except the variable name) but where did this `"` come from?  Its not mentioned in the manual as being required for globbing, nor in the parameter description of `sources`?

Also its only tested on `sources[0]` but then assumed on all sources below.

Reply:
> I know its not something you changed (except the variable name) but where did this " come from? Its not mentioned in the manual as being required for globbing, nor in the parameter description of sources?

This seems to be wrong. I think this was meant to detect
```
geany -g *
```
vs
```
geany -g "*"
```
where in the first case the shell expands `*` to the list of files Geany receives as its argument list while in the second case it receives `*` (but without the `"` which gets removed by the shell).

Not sure what we want and whether we actually want/need to support globbing by ourselves or whether what the shell does for us is sufficient already. I think the situation where this might be useful is when there are too many files matching the patterns so they exceed the max. number of command-line arguments.

Maybe we could just drop this check completely and always try to apply globbing (when `HAVE_GLOB_H` is defined).

Reply:
My preference (in that order)
1 Leave globbing to the shell
2 explicitly enable internal globbing with a separate parameter
3 Always apply globbing

I'm not sure if 1 leaves windows users behind, but if not I would greatly prefer that.